vital/types/local_geo_cs.cxx

```diff
@@ -136,18 +135,15 @@ ::update_camera(vital::metadata const& md,
 
   auto& md_sensor_location =
     md.find( vital::VITAL_META_SENSOR_LOCATION );
-  auto& md_sensor_altitude =
-    md.find( vital::VITAL_META_SENSOR_ALTITUDE );
-  if( md_sensor_location && md_sensor_altitude )
+  if( md_sensor_location )
```

Comment:
This can now be simplified: `if (auto& mdi = md.find(vital::VITAL_META_SENSOR_LOCATION))`